docs/source/programming_resources/blocks/Blocks-Tutorial.rst

```diff
@@ -24,52 +24,11 @@ to get acquainted with the Control System!
 
 |
 
-.. raw:: html
-
-   <h3>Table of Contents</h3>
-
-1. Introduction
-   
-   -  :doc:`The Control System <../shared/control_system_intro/The-FTC-Control-System>`
```

Comment:
If you're going to use badges to specify content-specific pages, we need to have some kind of notice on the main page (like this one blocks/Blocks-Tutorial.rst, or onbot_java/OnBot-Java-Tutorial.rst, or android_studio_java/Android-Studio-Tutorial.rst) stating that the links with the badges are content specific to the language, versus non-content-specific pages. Otherwise it's hard to know what the badges are for.